drivers/builtin/src/cipher_wrap.c

```diff
@@ -2119,136 +2119,6 @@ static const mbedtls_cipher_info_t null_cipher_info = {
 };
 #endif /* defined(MBEDTLS_CIPHER_NULL_CIPHER) */
 
-#if defined(MBEDTLS_NIST_KW_C)
```

Comment:
On line 51 of this file you can remove this block
```
#if defined(MBEDTLS_NIST_KW_C)
#include "mbedtls/nist_kw.h"
#endif
```

Reply:
Thanks for fixing this. Unfortunately I found another occurrence of `MBEDTLS_NIST_KW_C` at line 100 in this file, but I think it should be removed as well.